server/environment.ts

```diff
@@ -1,5 +1,5 @@
-import { Matches, MinLength, IsBoolean, Min, validate } from 'class-validator';
-import { Expose, Transform, Type, plainToClass } from 'class-transformer';
+import { Matches, MinLength, IsBoolean, validate, IsPort } from 'class-validator';
```

Comment:
Is `validate` used? :D

Reply:
It sure is! The validation function is under the class!